weather/weather.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import pyowm
+import os
```

Comment:
```suggestion
import os
import sys

default_place = "Gerbrunn,de"
```
see suggestion below